add_custom_view_and_activity_example/src/main/java/com/videoeditorsdk/android/app/KVideoEditorDemoActivity.kt

```diff
@@ -0,0 +1,160 @@
+package com.videoeditorsdk.android.app
+
+import android.app.Activity
+import android.content.ActivityNotFoundException
+import android.content.Intent
+import android.net.Uri
+import android.os.Build
+import android.os.Bundle
+import android.os.Environment
+import android.provider.MediaStore
+import androidx.annotation.RequiresApi
+import android.util.Log
+import android.widget.Button
+import android.widget.Toast
+import com.videoeditorsdk.android.app.custom.MyEditorBuilder
+import ly.img.android.pesdk.VideoEditorSettingsList
+import ly.img.android.pesdk.assets.filter.basic.FilterPackBasic
+import ly.img.android.pesdk.assets.font.basic.FontPackBasic
+import ly.img.android.pesdk.assets.frame.basic.FramePackBasic
+import ly.img.android.pesdk.assets.overlay.basic.OverlayPackBasic
+import ly.img.android.pesdk.assets.sticker.animated.StickerPackAnimated
+import ly.img.android.pesdk.assets.sticker.emoticons.StickerPackEmoticons
+import ly.img.android.pesdk.assets.sticker.shapes.StickerPackShapes
+import ly.img.android.pesdk.backend.model.EditorSDKResult
+import ly.img.android.pesdk.backend.model.constant.OutputMode
+import ly.img.android.pesdk.backend.model.state.LoadSettings
+import ly.img.android.pesdk.backend.model.state.VideoEditorSaveSettings
+import ly.img.android.pesdk.ui.activity.ExternalVideoCaptureBuilder
+import ly.img.android.pesdk.ui.model.state.*
+import ly.img.android.pesdk.ui.panels.item.PersonalStickerAddItem
+import ly.img.android.pesdk.ui.utils.PermissionRequest
+import ly.img.android.serializer._3.IMGLYFileWriter
+import java.io.File
+import java.io.IOException
+
+class KVideoEditorDemoActivity : Activity(), PermissionRequest.Response {
+
+    companion object {
+        const val VIDEO_EDITOR_SDK_RESULT = 1
+        const val CAMERA_AND_GALLERY_RESULT = 2
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        val openGallery = findViewById<Button>(R.id.openGallery)
+        openGallery.setOnClickListener {
+            openSystemGalleryToSelectVideo()
+        }
+
+        val captureVideo = findViewById<Button>(R.id.captureVideo)
+        captureVideo.setOnClickListener {
+            openSystemCameraToCaptureVideo()
+        }
+    }
+
+    // Create a empty new SettingsList and apply the changes on this referance.
+    // If you include our asset Packs and use our UI you also need to add them to the UI,
+    // otherwise they are only available for the backend (like Serialisation)
+    // See the specific feature sections of our guides if you want to know how to add your own assets.
```

Comment:
```suggestion
    // Create an empty new SettingsList and apply the changes to this reference.
    // If you include our asset packs and use our UI you also need to add them to the UI,
    // otherwise they are only available to the backend (e.g. serialization)
    // See the specific feature sections of our guides if you want to know how to add your own assets.
```